newIDE/app/src/MainFrame/index.js

```diff
@@ -1790,7 +1841,8 @@ class MainFrame extends React.Component<Props, State> {
               getCurrentTabIndex(this.state.editorTabs) === id;
             return (
               <ClosableTab
-                label={editorTab.name}
+                label={editorTab.label}
+                type={editorTab.type}
```

Comment:
This is not needed anymore, and you can also remove all the type on the editor tabs.